CMakeLists.txt

```diff
@@ -315,10 +315,12 @@ include(cmake/external_eigen.cmake)
 include(cmake/external_mkldnn.cmake)
 include(cmake/external_cldnn.cmake)
 
-if (NGRAPH_USE_PREBUILT_LLVM OR DEFINED LLVM_TARBALL_URL)
-    include(cmake/external_llvm_prebuilt.cmake)
-else()
-    include(cmake/external_llvm.cmake)
+if (NOT WIN32)
```

Comment:
Rather than `(NOT WIN32)` and `if (WIN32) else ...` which assume everything not windows is the same, you should condition on Linux and Macos here; better would be to have a variable for using LLVM that was true for Linux and Macos. Same for the other similar changes.